pkg/crc/api/api.go

```diff
@@ -68,8 +68,6 @@ func (api CrcApiServer) handleConnection(conn net.Conn) {
 			if handler, ok := api.handlers[req.Command]; ok {
 				result := handler(req.Args)
 				writeStringToSocket(conn, result)
-			} else {
-				writeStringToSocket(conn, fmt.Sprintf("Unknown command supplied: %s", req.Command))
```

Comment:
Since we are checking if the command is one of `start, stop, delete` we have a guarantee that Unknown command supplied is not going to happen...

Reply:
ah this could be moved to `Issue #949 Make cluster operation calls serve in order ` I believe?

Reply:
Actually rather than being removed, I'd just add a small `handleRequest` helper for
```
			if handler, ok := api.handlers[req.Command]; ok {
				result := handler(req.Args)
				writeStringToSocket(conn, result)
			} else {
				writeStringToSocket(conn, fmt.Sprintf("Unknown command supplied: %s", req.Command))
			}
```